plugins/uast-kotlin-fir/testData/legacy/PropertyInitializer.log.fir.txt

```diff
@@ -0,0 +1,8 @@
+UFile (package = )
+    UClass (name = TestPropertyInitializer)
```

Comment:
There is a noticeable difference: the lack of `UField` that corresponds to the backing field of `withSetter`. I'm suspecting `hasBackingField` in `FirLightClassBase.createField` is missing something. I can dig further on top of this, if the overall structure looks good enough.

Reply:
There is quite incomplete version of FirLightClasses so it is not super precise as far. So if you find that it  (hasBackingField for example) works not like UltraLight version please feel free to fix it or file an issue.